modules/codepipeline/templates/buildspec_deploy.yml

```diff
@@ -5,13 +5,13 @@ phases:
   install:
     runtime-versions:
       python: 3.7
+      golang: 1.14
     commands:
       - tf_version=$TERRAFORM_VERSION
       - wget https://releases.hashicorp.com/terraform/"$TERRAFORM_VERSION"/terraform_"$TERRAFORM_VERSION"_linux_amd64.zip
       - unzip terraform_"$TERRAFORM_VERSION"_linux_amd64.zip
       - mv terraform /usr/local/bin/
-      - curl -sSLo install.sh https://install.hclq.sh
```

Comment:
`hclq` is a very outdated tool. You can see [here](https://github.com/mattolenik/hclq) that it has not been updated since mid 2019. The issue relating to `hclq` not installing properly likely originates as a bug in the tool's install script for the latest "official" release. [This section](https://github.com/mattolenik/hclq#install-with-go) shows that you can use a different means of installing with go, that will obtain the most up-to-date version of the tool. This solves the intermittent installation failure bug.